share/spice/git_book_status/git_book_status.css

```diff
@@ -0,0 +1,25 @@
+.zci--git_book_status .gbstatus--header {
+    font-size: 1em;
```

Comment:
Let's remove this. `1em` is too small and we can use our `tx--##` classes if necessary